### PR TITLE
Update Objectives Statement for Clarity

### PR DESCRIPTION
**Pull request by germonprez**
Updating the objectives statement per Issue #292 

Signed-off-by: Matt Germonprez <germonprez@gmail.com>

### CODE REVIEW

**Comment by Nebrethar** [comment]
LGTM 👍

File: focus-areas/leadership/inclusive-leadership.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Leadership is central to project and community culture, and thus requires intent
 
 ## Objectives
 
-Signal to newcomers about who is welcome, and who can be successful. Ensure stable and consistent leadership that is representative of a project's diversity.
+Signal to newcomers that everyone is welcome and everyone can be successful. Ensure stable and consistent leadership that is representative of a project's diversity.
```
Comment (spotzz) [inline focus-areas/leadership/inclusive-leadership.md:14]: Rewording: Signal to newcomers that everyone is welcome and how to be successful within the project given any specified limitations listed (below/in section X/etc)

Comment (ElizabethN) [reply focus-areas/leadership/inclusive-leadership.md:14]: @germonprez am I ok to merge this then?

Comment (germonprez) [reply focus-areas/leadership/inclusive-leadership.md]: @ElizabethN yes, I made the changes.

**Pull request merged**